### ISSUE TITLE
Debug is false but appears as true

### ISSUE THREAD

**Issue by 1110815AdamSilva** [Silentscripter/laravel-prestashop-webservice#7]
Hey,

Sorry to create another issue but I'm still having problems. This image is what appears when I try to get customers.
![photo](https://cloud.githubusercontent.com/assets/3687887/18591851/448af442-7c2c-11e6-9034-0b744f0df6fa.png)

My debug is always set to false. Here is the function in the controller that get the data:
        public function bar()
        {
        $opt['resource'] = 'customers';
        $xml=$this->prestashop->get($opt);
        return view('admin.userinfo', ['data' => $xml]);
        }

Here is what I'm putting in the view:

`<p>{{$data}}</p>`

From what I've seen online, this is how to pass info from the controller to the view. Am I doing something wrong?

**Comment by Silentscripter** [comment]
Could you please provide more info on your code? What package version have you installed?

**Comment by Silentscripter** [comment]
I need more info to help you debugging this, the package version and the config file screen if possible, thanks.

**Comment by 1110815AdamSilva** [comment]
Hey, sorry, I was away all weekend and only managed to answer now. In the composer.json file:
"require": {
        "php": ">=5.6.4",
        "laravel/framework": "5.3.*",
        "protechstudio/laravel-prestashop-webservice": "^2.4"
    },

And by the config file screen, I'm going to assume it's the prestashop-webservice.php you want to see. It's attached.
![4a0f84f028457cba0344db4a5460b8ec](https://cloud.githubusercontent.com/assets/3687887/18627911/99c91bca-7e55-11e6-884a-9db0e6eb7735.png)

**Comment by Silentscripter** [comment]
Is the APP_DEBUG value present in .env or not?

**Comment by 1110815AdamSilva** [comment]
Yeah, it's set as true, but I thought that's the debug for the whole framework. The APP_DEBUG in .env and the APP_DEBUG in the prestashop-webservice.php don't work separately?

**Comment by 1110815AdamSilva** [comment]
So I solved the problem. I only figured out now that it's getting the global debug APP_DEBUG from the .env file. So to separate them and have the debug true for the rest of the code and have a "separate" debug for the PrestaShop Webservice, I created a new environment variable `PRESTASHOP_DEBUG=false` and then changed the prestashop-webservice.php to `'debug' => env('PRESTASHOP_DEBUG', false)`.
